src/app/zap-templates/zcl/data-model/chip/chip-types.xml

```diff
@@ -53,39 +41,37 @@ limitations under the License.
-    <type id="0xE2" description="UTC Time"                 name="utc_time"          size="4"  analog="true"   />
+    <type id="0xE2" description="UTC Time"                 name="utc"               size="4"  analog="true"   />
     <type id="0xE3" description="Epoch Microseconds"       name="epoch_us"          size="8"  analog="true"   />
     <type id="0xE4" description="Epoch Seconds"            name="epoch_s"           size="4"  analog="true"   />
     <type id="0xE5" description="System Time Microseconds" name="systime_us"        size="8"  analog="true"   />
-    <type id="0xE8" description="Cluster ID"               name="cluster_id"        size="2"  discrete="true" />
-    <type id="0xE9" description="Attribute ID"             name="attribute_id"      size="2"  discrete="true" />
+    <type id="0xE8" description="Cluster ID"               name="cluster_id"        size="4"  discrete="true" />
+    <type id="0xE9" description="Attribute ID"             name="attrib_id"         size="4"  discrete="true" />
     <type id="0xEA" description="Field ID"                 name="field_id"          size="4"  discrete="true" />
     <type id="0xEB" description="Event ID"                 name="event_id"          size="4"  discrete="true" />
-    <type id="0xEC" description="Command ID"               name="command_id"        size="1"  discrete="true" />
+    <type id="0xEC" description="Command ID"               name="command_id"        size="4"  discrete="true" />
     <type id="0xED" description="Action ID"                name="action_id"         size="1"  discrete="true" />
     <type id="0xEF" description="Transaction ID"           name="trans_id"          size="4"  discrete="true" />
     <type id="0xF0" description="Node ID"                  name="node_id"           size="8"  discrete="true" />
     <type id="0xF1" description="Vendor ID"                name="vendor_id"         size="2"  discrete="true" />
-    <type id="0xF2" description="Device Type ID"           name="device_type_id"    size="4"  discrete="true" />
+    <type id="0xF2" description="Device Type ID"           name="devtype_id"        size="4"  discrete="true" />
     <type id="0xF3" description="Fabric ID"                name="fabric_id"         size="8"  discrete="true" />
     <type id="0xF4" description="Group ID"                 name="group_id"          size="2"  discrete="true" />
     <type id="0xF5" description="Status Code"              name="status"            size="4"  discrete="true" />
     <type id="0xF6" description="Data Version"             name="data_ver"          size="4"  discrete="true" />
     <type id="0xF7" description="Event Number"             name="event_no"          size="8"  discrete="true" />
-    <type id="0xF8" description="Endpoint Number"          name="endpoint_id"       size="1"  discrete="true" />
+    <type id="0xF8" description="Endpoint Number"          name="endpoint_no"       size="2"  discrete="true" />
```

Comment:
Don't we usually call it Endpoint ID ?

Reply:
In the spec, it is named `endpoint-no`. I tried to follow the spec naming.